ui/app/components/tool-wrap.js

```diff
@@ -49,7 +43,6 @@ export default class ToolWrap extends Component {
   codemirrorUpdated(val, codemirror) {
     codemirror.performLint();
     const hasErrors = codemirror?.state.lint.marked?.length > 0;
-    this.data = val;
```

Comment:
instead, pass to parent callback `this.args.codemirrorUpdated` which handles setting `data` as the codemirror value